packages/leaders/scss/index-page.scss

```diff
@@ -33,3 +33,16 @@
     }
   }
 }
+
+.leaders {
```

Comment:
Does this also get applied to the "non-rounded" sites, e.g. Security Info Watch?
If so, this should only be applied to the "Pennwell" sites, not everyone.
You would need to create a separate SCSS file for this and import it only into the sites that need it.

Reply:
Or, is the rounding and box shadow disabled for those? I just need verification that by adding this it doesn't change the styling of the non-Pennwell sites.

Reply:
I wasn't aware that there was a different styling expectation for sites within the same repo. I'll look into a way to apply it to only specific sites, that wouldn't require replicating the css through each individual file.

Reply:
Can you update the primary screenshots and also provide a screen for SIW? Thanks!